src/main/kotlin/frc/robot/subsystems/wrist/Wrist.kt

```diff
@@ -0,0 +1,67 @@
+package frc.robot.subsystems.wrist
+
+import edu.wpi.first.math.geometry.Pose3d
+import edu.wpi.first.math.geometry.Rotation3d
+import edu.wpi.first.math.geometry.Translation3d
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.measure.Angle
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+import edu.wpi.first.wpilibj2.command.button.Trigger
+import org.littletonrobotics.junction.AutoLogOutput
+import org.littletonrobotics.junction.Logger
+import org.littletonrobotics.junction.mechanism.LoggedMechanism2d
+import org.littletonrobotics.junction.mechanism.LoggedMechanismLigament2d
+
+class Wrist(private val io: WristIO) : SubsystemBase() {
+    @AutoLogOutput private val mechanism = Mechanism2d(2.0, 3.0)
```

Comment:
You need to use a [`LoggedMechanism2d`](https://docs.advantagekit.org/data-flow/supported-types/#mechanisms-output-only).